code/game/objects/items/weapons/clown_items.dm

```diff
@@ -41,16 +42,26 @@
 	icon_state = "soapsyndie"
 	cleanspeed = 10 //much faster than mop so it is useful for traitors who want to clean crime scenes
 
+/obj/item/weapon/soap/vigilante
+	name = "cleaning rag"
+	desc = "All great things start with a little elbow grease."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "rag"
+	cleanspeed = 35
+	slippyness = 0
+
 /obj/item/weapon/soap/suicide_act(mob/user)
 	user.say(";FFFFFFFFFFFFFFFFUUUUUUUDGE!!")
 	user.visible_message("<span class='suicide'>[user] lifts [src] to their mouth and gnaws on it furiously, producing a thick froth! [user.p_they(TRUE)]'ll never get that BB gun now!")
 	new /obj/effect/particle_effect/foam(loc)
 	return (TOXLOSS)
 
 /obj/item/weapon/soap/Crossed(AM as mob|obj)
+	if(!slippyness)
+		return
 	if (istype(AM, /mob/living/carbon))
 		var/mob/living/carbon/M = AM
-		M.slip(4, 2, src)
+		M.slip(slippyness, slippyness, src)

```

Comment:
The two arguments on slip have been a meme for a long time. I figured out last year that literally only the larger of the two matter when I was doing my SLIP-SANITY pr. If youre stunned for 4 and weakened for 2, you're down for 4. Same vice-versa. 

As far as I can tell the stun itself is redundant, it doesn't do anything unless its longer than the weaken - in which case you might as well just set the weaken longer.